score.js

```diff
@@ -2,9 +2,16 @@ export let scoreContainer = document.getElementById("scoreContainer");
 
 export let score = 0;
 
-export function addScore() {
-  let scoreCounter = document.getElementById("scoreCounter"); // Jonathan
-  scoreCounter.innerHTML = " " + score++;
+let scoreCounter = document.getElementById("scoreCounter");
+
+export function addScore() { 
+  // scoreCounter.innerHTML = " " + score++;
+  scoreCounter.innerHTML = " " + (score++ + 1);
```

Comment:
If you test the counter without this, you'll notice that the amount of hits on screen will always lag behind. Meaning, it will say 0 even after 1 hit.